go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/bountylabs/go-metrics
```

Comment:
Do we also want to run `go mod tidy`?: https://blog.golang.org/migrating-to-go-modules

Seems to add a couple of dependencies on my local

Reply:
added them, let's see how things go